src/main/java/com/google/cloud/dataproc/templates/databases/SpannerToGCS.java

```diff
@@ -55,21 +63,21 @@ public void runTemplate() {
           String.format(
               "jdbc:cloudspanner:/projects/%s/instances/%s/databases/%s?lenient=true",
               projectId, instanceId, databaseId);
+      JdbcDialects.registerDialect(new SpannerJdbcDialect());
```

Comment:
I registered the dialect here. Note that this class does not even reference `JdbcPreparedStatement.java`. Or perhaps I'm misunderstanding how that class is used. Intellij could find no references to it.

Reply:
Yeah your template might not need JdbcPreparedStatement, but other one does.
It is not directly referenced in code, but invoked by spark automatically in the stack for SpannerToGCS template.

Reply:
I have undone the class removal and edits to the SpannerToGCS class, so this pr is just scopped to the GCSToSpanner job